test/topology_custom/test_long_join.py

```diff
@@ -27,16 +27,20 @@ async def test_long_join(manager: ManagerClient) -> None:
     await asyncio.gather(task)
 
 @pytest.mark.asyncio
-async def test_long_join_drop_wntries_on_bootstrapping(manager: ManagerClient) -> None:
+async def test_long_join_drop_entries_on_bootstrapping(manager: ManagerClient) -> None:
     """The test checks that join works even if expiring entries are dropped
        on the joining node between placement of the join request and its processing"""
-    s1 = await manager.server_add()
+    servers = await manager.servers_add(2)
     inj = 'topology_coordinator_pause_before_processing_backlog'
-    await manager.api.enable_injection(s1.ip_addr, inj, one_shot=True)
-    s2 = await manager.server_add(start=False,  config={
+    await manager.api.enable_injection(servers[0].ip_addr, inj, one_shot=True)
```

Comment:
I didn't notice this before, but there's a problem here

There's no guarantee that `servers[0]` will be the first server since you're using the concurrent `servers_add`, from looking at the code in test/pylib/scylla_cluster.py it is possible for the framework to schedule tasks in a way that makes `servers[1]` the initial seed -- so `servers[1]` could be the coordinator. But you're enabling the injection only on `servers[0]`. It should be enabled on both to guarantee that the coordinator hits it